python/pipeline/experiment.py

```diff
@@ -333,6 +333,73 @@ def plot_calibration_curve(self):
         return fig
 
 
+@schema
+class SurgeryType(dj.Lookup):
+    definition = """ # Surgery types used in experiment.Surgery
+    surgery_type                : varchar(64)         # Types of surgery performed on mice
+    """
+    contents = [
+        ['Cranial Window and Headbar'],
+        ['Headbar'],
+        ['Burr Hole and Suture'],
+        ['C-Section'],
+        ['Viral Injection'],
+        ['Electroporation']
+    ]
+
+
+@schema
+class SurgeryOutcome(dj.Lookup):
+    definition = """ # Surgery outcomes used in experiment.Surgery
+    surgery_outcome             : varchar(32)         # Possible outcomes of surgeries performed on mice
+    """
+    contents = [
+        ['Survival'],
+        ['Non-Survival'],
+    ]
+
+
+@schema
+class SurgeryQuality(dj.Lookup):
+    definition = """ # Surgery qualities used in experiment.Surgery
+    surgery_quality              : varchar(32)         # Possible qualities of surgeries performed on mice
+    """
+    contents = [
+        ['Good'],
+        ['Okay'],
+        ['Bad']
+    ]
+
+
+@schema
+class Surgery(dj.Manual):
+    definition = """ # List of surgeries performed on mice
+    -> mice.Mice
+    date                         : date                   # Date surgery was performed
+    ---
+    -> experiment.Person            
+    -> SurgeryOutcome
+    -> SurgeryType
+    -> SurgeryQuality
+    ketoprofen = null            : decimal(4,3)           # Amount of Ketoprofen given to mouse
+    weight = null                : decimal(5,2) unsigned  # Weight of mouse before surgery
+    notes                        : varchar(256)           # Notes on surgery
+    """
+
+
+@schema
+class SurgeryStatus(dj.Manual):
+    definition = """ # Updates to surgeries performed on mice
+    -> Surgery
+    timestamp                           : timestamp              # Timestamp of entry
+    ---
+    euthanized = "False"                : enum("True","False")
```

Comment:
you can also use type boolean and datajoint will internally convert it to tinyint (0/1 flag) and retrieve it as boolean (not 100% sure about the retrieval but it's probably more standard than an enum(False, True))